### PR TITLE
feat: diag tool proto parser and rpc service

### PR DESCRIPTION
**Pull request by zhangziheng01233**
- add proto parser
- add diag tool rpc service

### CODE REVIEW

File: python/openmldb_tool/diagnostic_tool/combined.proto
Comment (vagetablechicken) [inline python/openmldb_tool/diagnostic_tool/combined.proto:1]: combined.proto is needless, I'll fix it in this pr. Other improvements can be added in the future.

Comment (vagetablechicken) [reply python/openmldb_tool/diagnostic_tool/combined.proto:1]: use pb to parse for hint, and pb files will download when use

File: src/tablet/tablet_impl.cc [hunk 1]
```diff
@@ -3504,6 +3504,7 @@ void TabletImpl::GetTableFollower(RpcController* controller, const ::openmldb::a
     if (info_map.empty()) {
         response->set_msg("has no follower");
         response->set_code(::openmldb::base::ReturnCode::kNoFollower);
+        return;
```
Comment (vagetablechicken) [inline src/tablet/tablet_impl.cc]: create a new pr?

Comment (vagetablechicken) [reply src/tablet/tablet_impl.cc]: pr 3392, rebase this after pr merged

**Comment by vagetablechicken** [comment]
before: curl http://xx/xx/xx -H xx -d xxx, and need the ip port
now:
rpc ( install openmldb-tool[rpc] )
openmldb_tool --cluster=xxx rpc ns --hint
```


rpc CreateTable (CreateTableRequest) returns (GeneralResponse);

rpc DropTable (DropTableRequest) returns (GeneralResponse);

rpc ShowTablet (ShowTabletRequest) returns (ShowTabletResponse);

rpc ShowTable (ShowTableRequest) returns (ShowTableResponse);

rpc MakeSnapshotNS (MakeSnapshotNSRequest) returns (GeneralResponse);

rpc AddReplicaNS (AddReplicaNSRequest) returns (GeneralResponse);

rpc DelReplicaNS (DelReplicaNSRequest) returns (GeneralResponse);

rpc ShowOPStatus (ShowOPStatusRequest) returns (ShowOPStatusResponse);

rpc CancelOP (CancelOPRequest) returns (GeneralResponse);
```
openmldb_tool --cluster=xxx rpc ns ShowTable --field '{"show_all":true}'
openmldb_tool --cluster=xxx rpc tm ...
openmldb_tool --cluster=xxx rpc tablet1 ...

openmldb_tool --cluster=xxx rpc ns ShowTable --hint (need download proto py)
You should input json like this, ignore round brackets in the key and double quotation marks in the value: --field '{
    "(optional)name": "string",
    "(optional)db": "string",
    "(optional)show_all": "bool"
}'

File: python/openmldb_tool/diagnostic_tool/parser.py [hunk 2]
```diff
@@ -78,3 +93,81 @@ def zk_conn_err(self):
         checker = StatusChecker(conn)
         assert checker._get_components(show=False), "Failed to connect to zk"
         self.result += "\nSuccessfully checked zk connection. It may be caused by `Too many connections` in zk server. Please check zk server log."
+
+
+class ProtoParser:
```
Comment (vagetablechicken) [inline python/openmldb_tool/diagnostic_tool/parser.py]: remove it

**Review by dl239 — APPROVED** [review]

File: python/openmldb_tool/diagnostic_tool/parser.py (hunk 2 above)
Comment (vagetablechicken) [reply python/openmldb_tool/diagnostic_tool/parser.py]: bak commit https://github.com/zhangziheng01233/OpenMLDB/commit/71e5b13a83b55c3521b5d02462969fea112c7758

**Review by aceforeverd — APPROVED** [review]

**Pull request merged**